index.html

```diff
@@ -7428,7 +7428,7 @@ <h5>Presentational Role Inheritance</h5>
 					</tr>
 					<tr>
 						<th class="role-base-head" scope="row">Base Concept:</th>
-						<td class="role-base"> </td>
+						<td class="role-base"><a href="https://www.w3.org/TR/html5/grouping-content.html#the-search-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>search</code></a></td>
```

Comment:
respec uses the [data-cite](https://respec.org/docs/#data-cite) attribute to resolve this kind of thing.
```suggestion
						<td class="role-base"><a data-cite="html/grouping-content.html#the-search-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>search</code></a></td>
```

Reply:
Excellent, thank you